relay-general/src/protocol/metrics.rs

```diff
@@ -111,6 +111,24 @@ pub struct Metrics {
     /// This metric is measured in Sentry and reported in the javascript processing task.
     #[metastructure(field = "ms.processing.sourcemaps")]
     pub ms_processing_sourcemaps: Annotated<u64>,
+
+    /// Whether there has been a processing error that likely impacts the usefulness of an event.
+    ///
+    /// Example: malformed/unparseable debug information files.
+    ///
+    /// This metric is measured in Sentry and should be reported in all processing tasks.
+    #[metastructure(field = "error.processing")]
```

Comment:
To go in line with the others, should we rather call this `num.processing.errored` and `num.processing.failed`?

Reply:
it's a boolean though. maybe `flag`, `bool` or even `is.processing.failed`?

Reply:
Oh, right. `"flag"` also sounds good 👍   

Do we want to count errors maybe? I don't see much value in it, so the flag should be enough.

Reply:
I have an implementation of this linked in the PR description and I'd say it's too hard and may be misleading as we don't really know how much soft errors are swallowed by fatal errors.